motivaa-backend/motivaa-app/src/main/java/com/motivaa/boundary/UserController.java

```diff
@@ -1,22 +1,25 @@
 package com.motivaa.boundary;
 
-import com.motivaa.boundary.dto.CreateUserRequest;
-import com.motivaa.boundary.dto.CreateUserResponse;
-import com.motivaa.boundary.dto.FindUserByUuidRequestValidator;
-import com.motivaa.boundary.dto.FindUserByUuidResponse;
+import com.motivaa.boundary.dto.*;
```

Comment:
Should we import the whole package?

I know that this is easier and faster, especially if we need to use multiple classes or interfaces from the package. However, 
1. importing all public classes and interfaces in the package may lead to conflicts if there are multiple classes with the same name in different packages.

2. it could theoretically slow down compile time due to the increased number of classes being referenced. 

3. it is harder to track which specific classes are being used from the package, which can make maintenance and refactoring more challenging.

Reply:
I agree we should not import with wildcards. Changed it! FYI, I also had to change intellij settings as by default if you import more then 5 items from a package it changes to wildcard import automatically.